src/community/share-research-findings/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Share findings about your users
+description: Find out how to share information on how styles, components and patterns are performing
+section: Community
+theme: Ways to get involved
+layout: layout-pane.njk
+order: 2
+---
+
+<style>
+	code {
+		white-space : pre-wrap !important;
+	}
+</style>
+
```

Comment:
Should be able to remove these lines now @calvin-lau-sig7

The markdown code block should automatically set the styles since https://github.com/alphagov/govuk-design-system/pull/2563

Reply:
```suggestion
```